blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Announcing Wing Cloud
+description: We are not giving up on the cloud
```

Comment:
I'm not sure how many of our readers will feel like there is a strong movement to abandon the cloud.
I also think the description should be more positive if possible, so something like:
"Our vision for Wing Cloud" or "Helping developers to build above the clouds" or something else that is more inspirational